client/resteasy-client/src/test/scala/com/cerner/beadledom/client/resteasy/http/DefaultServiceUnavailableRetryStrategySpec.scala

```diff
@@ -26,19 +24,19 @@ class DefaultServiceUnavailableRetryStrategySpec extends FunSpec with Matchers w
 
     describe("#retryRequest") {
       describe("with status code 500 and executionCount 0") {
-        it should behave like retryableRequest(retryStrategy, 500, 0)
+        it should behave like nonRetryableRequest(retryStrategy, 500, 0)
       }
 
       describe("with status code 500 and executionCount 1") {
-        it should behave like retryableRequest(retryStrategy, 500, 1)
+        it should behave like nonRetryableRequest(retryStrategy, 500, 1)
       }
 
       describe("with status code 500 and executionCount 2") {
-        it should behave like retryableRequest(retryStrategy, 500, 2)
+        it should behave like nonRetryableRequest(retryStrategy, 500, 2)
       }
 
       describe("with status code 599 and executionCount 0") {
-        it should behave like retryableRequest(retryStrategy, 599, 0)
+        it should behave like nonRetryableRequest(retryStrategy, 599, 0)
       }
```

Comment:
Probably need tests for `describe("with status code 503 and executionCount 2")` and `describe("with status code 503 and executionCount 3")`

Reply:
https://github.com/cerner/beadledom/pull/96/commits/b682f04ded7a42813b08f18966bb3730bb2ca729